cumulusci/tasks/bulkdata/snowfakery_utils/subtask_configurator.py

```diff
@@ -29,6 +29,15 @@ def data_generator_opts(self, working_dir, recipe_options=None, *args, **kwargs)
         parts = name.rsplit("_", 1)
         batch_size = int(parts[-1])
 
+        # The pid is the tool that ensures that every uniqueid is
+        # unique even across different processes.
+        #
+        # big_ids ensures that the pid is incorporated into the ids.
+        plugin_options = {
+            "pid": str(wd.index),
+            "big_ids": "True",
```

Comment:
Should we not use a name relating to pid?

Reply:
@jofsky yes we could consider renaming in the future, although this name is documented for now.